src/test/java/com/stripe/model/AccountTest.java

```diff
@@ -21,14 +19,12 @@
 import static org.mockito.Mockito.*;
 
 public class AccountTest extends BaseStripeMockTest {
-	public static final String RESOURCE_ID = "acct_123";
-
 	@Test
 	public void testDeserialize() throws StripeException, IOException {
-		String json = getResourceAsString("account.json");
+		String json = getResourceAsString("/api_fixtures/account.json");
```

Comment:
Rather than deserializing from an embedded JSON fixture, I'd rather we use stripe-mock whenever possible.

Reply:
I disagree partially because stripe-mock return non Custom accounts which won't deserialize a lot of the more complex sub-elements like `legal_entity` and `external_accounts`

Reply:
I'm happy to do both but that deserialization via stripe-mock happens in most of the other tests since we retrieve an account before reject/update/etc.